### PR TITLE
React-multiple-components - Lindsay Jin

### PR DESCRIPTION
**Pull request by lindsay-jin**
Screen Shot 2024-03-22 at 3 52 13 PM

### CODE REVIEW

**Review by RobertGardner — COMMENTED** [review]
Superb!

File: react-multiple-components/my-app/src/Indicators.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+type Prop = {
+  count: number;
+  index: number;
+};
+
+export function Indicators({ count }: Prop) {
+  const rowOfButtons: JSX.Element[] = [];
+  for (let i = 0; i < count; i++) {
+    rowOfButtons.push(<button key={i}>{i}</button>);
```
Comment (RobertGardner) [inline react-multiple-components/my-app/src/Indicators.tsx]: Now you need to change the background color of the button that matches `index` to be light blue. You can do that by comparing `i` with `index`.

**Review by RobertGardner — COMMENTED** [review]
Great! One comment for something that you'd normally do in Phase 2.

**Comment by lindsay-jin** [comment]
Kapture 2024-03-23 at 10 19 01

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**